plugins/techdocs/src/home/components/DocsTable.tsx

```diff
@@ -128,3 +120,6 @@ export const DocsTable = ({
     </>
   );
 };
+
+DocsTable.columns = columnFactories;
```

Comment:
This assignment also seems to be causing issues, but because `DocsTable` is exported as an extension in `plugin.ts`. Not too sure what to do about it atm, perhaps creating an explicit exported type for the extension could help, and it's also a bit curious that this component both has the columns and actions as well as being used as an extension directly. Perhaps this shouldn't be exported as an extension at all but use the pattern that the catalog plugin does for the table page customization @backstage/techdocs-core?

Reply:
I opted to remove these from being exported through `DocsTable` and only exposing them through `EntityListDocsTable` instead as that component is being exported not as an extension (following the catalog customization pattern)